static_analyzer/clustering/names/__init__.py

```
@@ -0,0 +1,50 @@
+    "Unit",
+    "draft_scope",
+    "draft_tree",
+    "replay",
+    "role_words_for",
+    "stem",
+    "tokenize",
+    "unit_position",
+    "units_from_graph",
+    "units_from_graphs",
+    "walk",
+]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)